code/modules/events/abductor.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/abductor
+	name = "Abductors"
+	typepath = /datum/round_event/abductor
+	weight = 5
+	max_occurrences = 1
+
+	earliest_start = 180000 // 30 min
+
+	gamemode_blacklist = list("nuclear","wizard","cult","gang","revolution","abduction","blob")
```

Comment:
Sorry I didn't catch that this wasn't already a thing, I thought blobs were already kept out of blob mode to avoid double dipping.

I'm not really assuming anything, I'm just saying I don't see the threat of abductors screwing up those roundtypes.